tests/test_sql_utils.py

```diff
@@ -0,0 +1,291 @@
+import unittest
+
+from streaming_jupyter_integrations.sql_utils import is_ddl, is_dml, is_query
+
+
+# Some of the following examples are taken from Flink SQL documentation
+# https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sql/overview/
+class TestSqlUtils(unittest.TestCase):
+    def test_simple_select(self):
+        simple_select = "   SELECT   * FROM\n\n  myTable\n\t WHERE   someField\r = \r\nsomeOtherField \n\n "
+        assert not is_ddl(simple_select)
```

Comment:
Any reasons we're mixing `assert` with `assertTrue` in the tests ?

Reply:
I've started with plain `assert` as it is what's in other tests. However when I decided to use `with self.subTest` I started using `assertTrue` in those clauses. Probably should be changed to `assertTrue`/`assertFalse` everywhere